.github/workflows/build-windows.yml

```
@@ -47,17 +47,20 @@ jobs:
           fi
 
+      - name: Set up pnpm
+        if: ${{ inputs.should_run }}
+        uses: pnpm/action-setup@v4
+        with:
+          version: 10.33.0
+          run_install: false
+
       - name: Set up Node
         if: ${{ inputs.should_run }}
         uses: actions/setup-node@v3
```

actionlint (1.7.12)

[error] the runner of "actions/setup-node@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)